_posts/2017-09-25-bagwell-award-2017.md

```diff
@@ -0,0 +1,39 @@
+---
+category: news
+title: "Josh Suereth receives community award"
+---
+
+We are pleased to announce that the Phil Bagwell Memorial Scala Community Award for 2017 has been awarded to [Josh Suereth](https://twitter.com/jsuereth).  The award was presented in England's Lake District at [Scala World 2017](https://scala.world).
+
+Josh has been a prominent member of the Scala community ever since the early
+days of the language's commercial adoption.  He is known to Scala users as:
+
+* author of [_Scala in Depth_](https://www.manning.com/books/scala-in-depth)
+  and co-author of [_sbt in Action_](https://www.manning.com/books/sbt-in-action)
+* maintainer of the [sbt](http://www.scala-sbt.org) build tool
+  for several years at Typesafe (the company now known as Lightbend)
+* leader or contributor to numerous other open source Scala projects
+  including scala-arm, pickling, and assorted sbt plugins
+* cohost of the [Scalawags podcast](http://scalawags.tv)
+  (and its predecessor, The Scala Types)
+* a friendly and tireless conference-goer and presenter (and
+  organizer of the Scala Lift Off unconference in Virginia in 2009)
+* a positive force on nearly every online medium devoted to
+  Scala -- mailing lists, chat rooms, Twitter, you name it --
+  answering any and all questions with tact and humor
+
+Here's a
+[three minute video](https://www.youtube.com/watch?time_continue=2&v=0sLixaObVwA)
```

Comment:
Not sure, but should the URL omit the `time_continue` parameter?

Reply:
oops, fixed